app/pages/admin/project-status.jsx

```diff
@@ -124,17 +124,23 @@ class ProjectStatus extends Component {
       .catch(error => this.setState({ error }));
   }
 
+  setFeatured(project, value) {
+    project.update({ featured: value.checked });
+    return project.save()
+      .catch(error => this.setState({ error }));
+  }
+
   handleFeaturedProjectChange({ target }) {
     const { featured, project } = this.state;
     if (featured) {
       featured.update({ featured: false });
-      featured.save()
+      return featured.save()
         .then(() => this.setState({ featured }))
-        .catch(error => this.setState({ error }));
+        .catch(error => this.setState({ error }))
+        .then(this.setFeatured(project, target));
+    } else {
+      return this.setFeatured(project, target);
```

Comment:
There must always be a featured project in the database. If you get here, something is wrong.

Reply:
Panoptes allows no featured projects being set, so technically it's not wrong, I suppose.